integration/env/common.go

```diff
@@ -31,6 +44,34 @@ func init() {
 	if circleSHA == "" {
 		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarCircleSHA))
 	}
+	testedVersion = os.Getenv(EnvVarTestedVersion)
+	if testedVersion == "" {
+		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarTestedVersion))
+	}
+	token = os.Getenv(EnvVarGithubBotToken)
+	if token == "" {
+		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarGithubBotToken))
+	}
+
+	var err error
+	params := &framework.VBVParams{
+		Component: "chart-operator",
+		Provider:  "aws",
+		Token:     token,
+		VType:     TestedVersion(),
+	}
+	versionBundleVersion, err = framework.GetVersionBundleVersion(params)
```

Comment:
This is aligned to `aws-operator` right now. I want to move this stuff out of env in the future, possibly into setup?